eureka2-core/src/main/java/com/netflix/eureka2/transport/EurekaTransports.java

```diff
@@ -89,7 +90,7 @@
     static final String INTEREST_ENVELOPE_TYPE = "com.netflix.eureka2.protocol.interest.InterestMessage";
 
     static final Class<?>[] INTEREST_PROTOCOL_MODEL = {
-            InterestRegistration.class, UnregisterInterestSet.class, Heartbeat.class,
+            InterestRegistration.class, UnregisterInterestSet.class, Heartbeat.class, DeltaDTO.class,
```

Comment:
Only top level classes must be registered here. Do we send DeltaDTO as a top level message?